src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -566,6 +561,21 @@ trait BeamHelper extends LazyLogging {
         scenario.getPopulation.removePerson(personId)
       }
 
+      val csvWriter = new FileWriter("householdVehicles.csv", true)
+      csvWriter.write("vehicleId,vehicleType,householdId\n")
+
+      scenario.getHouseholds.getHouseholds.values.asScala.foreach { householdId =>
+        householdId.getVehicleIds.asScala.foreach { vehicle =>
+          csvWriter.write(
+            beamServices.privateVehicles.get(vehicle).get.id + "," +
```

Comment:
You can reduce the reads by storing the `get` in a variable. 

Also, you do a `map...getOrElse` in the second position, but `get` in the first. This should be consistent - You could even just `fold` and output both together?